applets/notification_area/main.c

```diff
@@ -416,44 +406,27 @@ static void
 na_tray_applet_realize (GtkWidget *widget)
 {
 	NaTrayApplet *applet = NA_TRAY_APPLET (widget);
-	MatePanelAppletOrient orient;
-	char *filename;
-	GError *error;
 	GtkActionGroup* action_group;
-	gchar* ui_path;
 
 	if (parent_class_realize)
 		parent_class_realize (widget);
 
 	action_group = gtk_action_group_new ("NA Applet Menu Actions");
 	gtk_action_group_set_translation_domain (action_group, GETTEXT_PACKAGE);
 	gtk_action_group_add_actions (action_group, menu_actions, G_N_ELEMENTS(menu_actions), applet);
-	ui_path = g_build_filename (NOTIFICATION_AREA_MENU_UI_DIR, "notification-area-menu.xml", NULL);
-	//mate_panel_applet_setup_menu_from_resource (MATE_PANEL_APPLET (applet),
-	//					      NA_RESOURCE_PATH "notification-area-menu.xml",
-	//					      action_group);
-	mate_panel_applet_setup_menu_from_file (MATE_PANEL_APPLET(applet), ui_path, action_group);
-	g_free (ui_path);
+	mate_panel_applet_setup_menu_from_resource (MATE_PANEL_APPLET (applet),
```

Comment:
Why not reverting this where you comment out the code?

Reply:
I have removed the original changes where I undid the GResource changes, so this commit does not exist anymore.